.github/workflows/test.yml

```diff
@@ -8,12 +8,26 @@ on:
             - "releases/**"
         tags:
             - "*"
+        paths-ignore:
+            - "**.md"
+            - "examples/**"
+            - "**/doc/**"
+            - "data-processing-lib/doc/**"
+            - "**/.gitignore"
+            - "**/.dockerignore"
     pull_request:
         branches:
             - "dev"
             - "releases/**"
+        paths-ignore:
+            - "**.md"
+            - "examples/**"
+            - "doc/**"
+            - "data-processing-lib/doc/**"
```

Comment:
as above

Reply:
I changed to `**/doc/**` should catch kfp too now I believe. Thanks.